paddle/fluid/operators/deformable_conv_cu.h

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
If these kernels are used by CPU and GPU, maybe the naming `deformable_conv_**cu**` for this file is not appropriate?

Reply:
Done